pss/pss_test.go

```diff
@@ -1836,11 +1802,30 @@ func newServices(allowRaw bool) adapters.Services {
 		params.MaxRetries = 1000
 		params.RetryExponent = 2
 		params.RetryInterval = 1000000
-		kademlias[id] = network.NewKademlia(id[:], params)
+		kademlias[id] = network.NewKademlia(bzzKey, params)
 		return kademlias[id]
 	}
-	return adapters.Services{
-		protocolName: func(ctx *adapters.ServiceContext) (node.Service, error) {
+	return map[string]simulation.ServiceFunc{
+		"bzz": func(ctx *adapters.ServiceContext, bucket *sync.Map) (s node.Service, cleanup func(), err error) {
```

Comment:
Please don't move blocks of code when making changes in the future. It's easier to compare if you leave them where they are, then we can always reorganize the blocks separately.